### ISSUE TITLE
Travix problem with 4.0.0-preview5

### ISSUE THREAD

**Issue by ciscoheat** [haxetink/tink_syntaxhub#11]
This started to appear, crashing all targets on all builds: https://travis-ci.org/ciscoheat/buddy/jobs/448948009#L497

```
/home/travis/haxe/lib/tink_syntaxhub/0,3,6/src/tink/syntaxhub/FrontendContext.hx:138: characters 14-19 : Variable requires type-hint or initialization
/home/travis/haxe/lib/tink_await/0,4,0/src/tink/await/Await.hx:10: lines 10-88 : Defined in this class
/home/travis/haxe/lib/tink_syntaxhub/0,3,6/src/tink/SyntaxHub.hx:27: characters 5-31 : Class<tink.syntaxhub.FrontendContext> has no field resetCache
/home/travis/haxe/lib/tink_syntaxhub/0,3,6/src/tink/SyntaxHub.hx:28: characters 28-52 : Class<tink.syntaxhub.FrontendContext> has no field findType
```

**Comment by ciscoheat** [comment]
Easy fix in `FrontendContext.hx` line 138:

`static var cache : Map<String, TypeDefinition>;`

**Comment by kevinresol** [comment]
It is already fixed but I can't make a release due to #10

**Comment by back2dos** [comment]
Should work now.
